README.md

```diff
@@ -124,27 +128,149 @@ a proxy tool such as [mitmproxy](https://mitmproxy.org), or even just
 a script.
 
 A particularly simple way to proxy requests to a private bitcoind node
-is to make use of the [`corsproxy`](https://www.npmjs.com/package/corsproxy)
-npm module. Instructions to install and run the module are on its
-[home page](https://www.npmjs.com/package/corsproxy).
+is to make use of [`nginx`](https://nginx.org/). Instructions to install
+and run the program are on its [download page](https://nginx.org/en/download.html).
 
-Explicitly, install `corsproxy` with
+Explicitly, install `nginx` with
 
+```bash
+# MacOS
+brew install nginx
+
+# Debian Linux
+sudo apt install nginx
 ```
-$ npm install -g corsproxy
+
+Copy the server conifiguration file, `bitcoind.proxy`, to the appropriate location with the following
+commands. Note, these commands assume that you are in the base `caravan` directory. An example configuration
+file is included with the `caravan` source code called `bitcoind.proxy` which will, by defualt, enable a mainnet
+proxy. The testnet proxy is included, but is commented out.
+
+```bash
+# MacOS
+mkdir -p /usr/local/etc/nginx/sites-available
+cp bitcoind.proxy /usr/local/etc/nginx/sites-available/
+ln -s /usr/local/etc/nginx/sites-available/bitcoind.proxy /usr/local/etc/nginx/servers/bitcoind
+
+# Debain Linux
+sudo mkdir -p /etc/nginx/sites-available
+sudo cp bitcoin.proxy /etc/nginx/sites-available/
+sudo ln -s /etc/nginx/sites-available/bitcoin.proxy /etc/nginx/sites-enabled/bitcoind
 ```
 
-and then launch corsproxy
+check that the file is copied correctly
 
+```bash
+$ nginx -t
+nginx: the configuration file /usr/local/etc/nginx/nginx.conf syntax is ok
+nginx: configuration file /usr/local/etc/nginx/nginx.conf test is successful
 ```
-$ corsproxy
+
+Start `nginx`
+
+```bash
+# MacOS
+brew services start nginx
+# or if nginx is already running
+brew services reload nginx
+
+# Debain Linux
+sudo systemctl start nginx
+# or if nginx is already running
+sudo systemctl restart nginx
+```
+
+On MacOS, starting the `nginx` daemon will prompt a popup window asking if you want `ngingx`
+to allow incoming network connections, which you will want to allow.
+
+Test the different ports:
+
+```bash
+# Test that bitcoin rpc is functioning correctly
+curl --user my_uname --data-binary \
+'{"jsonrpc": "1.0", "id":"curltest", "method": "getblockcount", "params": [] }' \
+-H 'content-type: text/plain;' http://127.0.0.1:8332
```

Comment:
We should put the expected result in here so users know what they should be looking for.  Something like:

```
{"result":668255,"error":null,"id":"curltest"}
```

Reply:
Added the example output.